airbyte-integrations/connectors/source-harvest/unit_tests/integration/test_projects.py

```diff
@@ -0,0 +1,394 @@
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+
+import json
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+from unittest import TestCase
+
+import freezegun
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.test.catalog_builder import CatalogBuilder
+from airbyte_cdk.test.entrypoint_wrapper import read
+from airbyte_cdk.test.mock_http import HttpMocker, HttpResponse
+from airbyte_cdk.test.state_builder import StateBuilder
+
+
+# Add parent directory to path to allow imports
+sys.path.insert(0, str(Path(__file__).parent.parent))
```

Comment:
This is interesting. Do you know why this needed to be added and what happens if it is removed.

In other connectors that used custom components in `conftest.py` we add: `sys.path.append(str(_SOURCE_FOLDER_PATH))  # to allow loading custom components`.

Granted harvest doesn't have custom components, so I was curious what this line is doing